macros/generate_schema_name.sql

```diff
@@ -1,14 +1,10 @@
-{% macro generate_schema_name(custom_schema_name, node) -%}
-
-    {%- set default_schema = target.schema -%}
-    {%- if custom_schema_name is none -%}
-
-        {{ default_schema }}
+{#
+    This macro has been modified to work with the variables set in the dbt_project.yml file.
+    See https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas for the original macro.
+#}
 
-    {%- else -%}
-
-        {{ custom_schema_name | trim }}
+{% macro generate_schema_name(custom_schema_name, node) -%}
```

Comment:
This is how the macro looks in CC. I'm happy to change it back to the default from dbt. I think our vars should prevent this from doing things we don't want.